src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,176 @@
+package duke;
+
+import java.util.List;
+
+/**
+ * Represents the user interface and displays messages for Duke.
+ */
+public class Ui {
```

Comment:
Since this class does not have any state, would it more appropriate for it's methods to be static?